src/pages/WorkloadDetails/WorkloadInfo.tsx

```diff
@@ -223,14 +270,55 @@ class WorkloadInfo extends React.Component<WorkloadInfoProps, WorkloadInfoState>
       <>
         Pods ({pods.length}){' '}
         {validationChecks.hasPodsChecks
-          ? getValidationIcon(
+          ? getWorkloadValidationIcon(
               pods.map(a => a.name),
               'pod'
             )
           : undefined}
       </>
     );
 
+    const istioConfigItems = this.state.workloadIstioConfig ? toIstioItems(this.state.workloadIstioConfig) : [];
+    let istioTabTitle: JSX.Element | undefined;
+    let istioConfigIcon = undefined;
+    if (this.state.workloadIstioConfig?.validations) {
+      const typeNames: { [key: string]: string[] } = {};
+      if (this.state.workloadIstioConfig.validations['gateway']) {
```

Comment:
It is probably style, but do you think it is a way to do this if/else statements shorter not repeating code?
I guess that using dynamic access to objects of JS/TS could help reduce the number of lines of code.
```this.state.workloadIstioConfig["envoyFilters"].forEach()``` could work.

Reply:
Yes, good comment, on this side I think this would work.